master/pkg/searcher/search_method.go

```diff
@@ -122,3 +121,11 @@ func (defaultSearchMethod) trialExitedEarly( //nolint: unused
 	context, model.RequestID, workload.ExitedReason) ([]Operation, error) {
 	return []Operation{Shutdown{Failure: true}}, nil
 }
+
+func sumTrialLengths(us map[model.RequestID]model.PartialUnits) model.PartialUnits {
```

Comment:
does this take into account invalidHP-exited trials?

Reply:
yeah they're deleted from the list elsewhere.